src/metamodels/crosscat.py

```diff
@@ -33,8 +36,8 @@
 import bayeslite.guess as guess
 import bayeslite.metamodel as metamodel
 import bayeslite.weakprng as weakprng
-import crosscat_generator_schema
-import crosscat_theta_validator
+from bayeslite.metamodels import crosscat_generator_schema
```

Comment:
Why do you prefer absolute_import here? It seems unrelated to the rest of the changes, so I might prefer to revert.